src/common/image.js

```diff
@@ -23,11 +23,11 @@ const baseStyleSheet/*:StyleSheet*/ = Style.createSheet
   );
 
 export const view =
-  (key/*:string*/, styleSheet/*:StyleSheet*/)/*:(model:Model, address:Address<Action>, contextStyle?:ContextStyle) => DOM*/ =>
-  ( model/*:Model*/
-  , address/*:Address<Action>*/
+  (key:string, styleSheet:StyleSheet)/*:(model:Model, address:Address<Action>, contextStyle?:ContextStyle) => DOM*/ =>
+  ( model:Model
+  , address:Address<Action>
   , contextStyle/*?:ContextStyle*/
```

Comment:
Type annotation here is still commented, presumably because script does not considers optional argument syntax - `contextStyle?:ContextStyle`.